src/Summoner/Project.hs

```diff
@@ -115,8 +115,11 @@ generateProject projectName Config{..} = do
         -- Make a commit and push it.
         "git" ["add", "."]
         "git" ["commit", "-m", "Create the project"]
-        "git" ["push", "-u", "origin", "master"]
-
+        unless noUpload ( do
+            "hub" $ ["create", "-d", settingsDescription, settingsOwner <> "/" <> settingsRepo]
```

Comment:
I noticed that the line with the same code was not removed a few lines before. As we only need to create the repo if `noUpload` option is off, you can remove that line above.

https://github.com/kowainik/summoner/blob/52e2303862877ad82ae586210b0b591e9464a649/src/Summoner/Project.hs#L113-L114